### PR TITLE
ref #114: Add cache clear command

### PR DESCRIPTION
**Pull request by UlrichKu**
| Q             | A
| ------------- | ---
| Branch        | 6.2.x
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Fixed issues  | chameleon-system/chameleon-system#114
| License       | MIT

### CODE REVIEW

File: src/CmsCacheBundle/Command/ClearChameleonCacheCommand.php [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        $this->cache = $cache;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this->setName('chameleon_system:cache:clear')
+            ->setDescription('Clears the chameleon cache.');
```
Comment (umulmrum) [inline src/CmsCacheBundle/Command/ClearChameleonCacheCommand.php]: "chameleon" --> "Chameleon"

File: src/CmsCacheBundle/Command/ClearChameleonCacheCommand.php [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    /**
+     * @var CacheInterface
+     */
+    private $cache;
+
+    /**
+     * @param CacheInterface $cache
+     * @param null|string    $name
+     *
+     *  @throws LogicException When the command name is empty
```
Comment (umulmrum) [inline src/CmsCacheBundle/Command/ClearChameleonCacheCommand.php]: Double space before "@throws".

**Comment by umulmrum** [comment]
This is a feature, not a fix - that's OK here as the feature is completely independent, but please adjust the PR description :-)

File: src/CmsCacheBundle/Command/ClearChameleonCacheCommand.php (hunk 1 above)
Comment (UlrichKu) [reply src/CmsCacheBundle/Command/ClearChameleonCacheCommand.php]: ok

File: src/CmsCacheBundle/Command/ClearChameleonCacheCommand.php (hunk 2 above)
Comment (UlrichKu) [reply src/CmsCacheBundle/Command/ClearChameleonCacheCommand.php]: ok

**Comment by umulmrum** [comment]
Thanks @UlrichKu

**Pull request merged**